### PR TITLE
Mark the old Death Note addons as imcompatible

### PR DESCRIPTION
**Pull request by xopez**
- Roles are hardcoded
- In TTT2 all addons print only detective roles
- The new addon prints all roles except the own teamrole no matter for which team they play
- Hidden roles for the user are still hidden for him so there is no reveal

### CODE REVIEW

**Comment by TimGoll** [comment]
LET THE CODE BEAUTYFIERS FIGHT!

File: gamemodes/terrortown/gamemode/server/sv_addonchecker.lua [hunk 1]
```diff
@@ -184,25 +184,40 @@ addonChecker.curatedList = {
 	},
 	["1729451064"] = { -- TTT - More Melee Weapons by Solrob
 		reason = "Overwrites stock TTT2 crowbar which chauses problems with doors and ttt_map_settings entity.",
```
Comment (TimGoll) [inline gamemodes/terrortown/gamemode/server/sv_addonchecker.lua:186]: not your fault, but could you fix this typo while you're at it? Thanks!

Comment (xopez) [reply gamemodes/terrortown/gamemode/server/sv_addonchecker.lua]: fixed, had to remerge from origin, hope it didn't break anyting

**Comment by TimGoll** [comment]
looks good! thank you

**Comment by TimGoll** [comment]
FYI, in our workflow the reviewer closes the conversations. Doesn't matter in this instance, but just so you are aware

**Comment by xopez** [comment]
Thanks, good to know. I will follow that in future

**Review by ZenBre4ker — APPROVED** [review]

**Pull request merged**